docs/script-runner-addon.md

```diff
@@ -0,0 +1,28 @@
+# Script Runner Addon Guide
+
+Script Runner lets you run a bash script on all your cluster nodes with one command. It uploads the bash script to each node then execute. The script will receive two arguments upon execution:
+1) Node group, for example (master, worker, etcd)
+2) Cluster info JSON config, which is similar to the hetzner-kube config found in `~/.hetzner-kube/config.json`
+
+### Example
+hetzner-kube Command:
+```bash
+hetzner-kube cluster addon install --name demo script-runner /path/to/script.sh  
+```
+Example script.sh:
+```bash
+

```

Comment:
Bash script should start with:
```
#!/bin/bash
```
this should be omitted (than specify on the doc that should not be included) or is missing here (than add to the sample)?